pkg/crc/api/api_http_test.go

```diff
@@ -265,7 +265,7 @@ var testCases = []testCase{
 	// version
 	{
 		request:  get("version"),
-		response: jSon(fmt.Sprintf(`{"CrcVersion":"%s","CommitSha":"%s","OpenshiftVersion":"%s"}`, version.GetCRCVersion(), version.GetCommitSha(), version.GetBundleVersion())),
+		response: jSon(fmt.Sprintf(`{"CrcVersion":"%s","CommitSha":"%s","OpenshiftVersion":"%s","PodmanVersion":"%s"}`, version.GetCRCVersion(), version.GetCommitSha(), version.GetBundleVersion(), version.GetPodmanVersion())),
```

Comment:
There were some discussions to give `OpenShiftVersion`/`PodmanVersion` more descriptive names, as these are the versions of the bundles supported by the binary, not necessarily the version currently being used.

Reply:
it'll make more sense when multiple bundles can be used with `crc` for now only the supported bundle can be used with the released `crc` binary, or did we remove that restriction?

Reply:
The restriction is still in place